crates/core/src/config/read.rs

```diff
@@ -142,20 +142,26 @@ mod tests {
             ),
         ]);
         assert_eq!(
-            InputPartitions.parse_value(&options).unwrap().to::<usize>(),
+            { let value: usize = InputPartitions.parse_value(&options).unwrap().into(); value },
```

Comment:
try to avoid the pattern. use sth like this before the assert to keep code clean. `actual` can be reused to save naming hassle. applies to all other similar changes

```rust
        let actual: usize = InputPartitions.parse_value(&options).unwrap().into();
```